.editorconfig

```diff
@@ -0,0 +1,14 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
Is this for your editor? If so, add to .gitignore

Reply:
editorconfig is a very popular way to keep a consistent coding style across IDEs and editors and should not be in a gitignore as a lot of popular repositories show (e.g.  react-bootstrap)